content/applications/finance/accounting/fiscal_localizations/localizations/mexico.rst

```diff
@@ -496,6 +496,14 @@ payment, 15 days, 21 days, all falling within the current month).
 Payments
 --------
 
+.. warning::
+   - When creating a payment in MXN for an invoice in USD, the payment **must** be created using the
+     'Register Payment' button on the invoice view and not separately as a payment. Otherwise, the
+     payment CFDI will not be correctly generated.
+   - As such, a payment in MXN cannot be used to pay multiple invoices in USD. Rather, the payment
+     should be separated into multiple payments, and each payment created using the 'Register Payment'
+     button on its corresponding invoice.
```

Comment:
```suggestion
.. warning::
   - When making a payment in MXN for an invoice in USD, the payment must be created using the
     :guilabel:`Register Payment` button **on the invoice view** and not separately as a payment.
     Otherwise, the payment CFDI is not correctly generated.
   - As such, a payment in MXN cannot be used to pay multiple invoices in USD. Rather, the payment
     should be separated into multiple payments created using the :guilabel:`Register Payment`
     button on the corresponding invoices.
```